mungegithub/github/github.go

```diff
@@ -569,6 +569,28 @@ func (config *Config) NewIssue(title, body string, labels []string, owner string
 	return obj, nil
 }
 
+// GetBranchCommits gets recent commits for the given branch.
+func (config *Config) GetBranchCommits(branch string) ([]*github.RepositoryCommit, error) {
+	commits := []*github.RepositoryCommit{}
+	page := 0
+	for {
+		commitsPage, response, err := config.client.Repositories.ListCommits(
+			config.Org, config.Project,
+			&github.CommitsListOptions{ListOptions: github.ListOptions{PerPage: 100, Page: page}, SHA: branch})
+		config.analytics.ListCommits.Call(config, response)
+		if err != nil {
+			glog.Errorf("Error reading commits for branch %s: %v", branch, err)
+			return nil, err
+		}
+		commits = append(commits, commitsPage...)
+		if response.LastPage == 0 || response.LastPage <= page || len(commits) > 200 {
```

Comment:
Why 200? Why not just stop at 100? Seems like you shouldn't even need a loop here.